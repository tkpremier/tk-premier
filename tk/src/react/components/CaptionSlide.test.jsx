import React from 'react';
import SlideCaption from './CaptionSlide';
import renderer from 'react-test-renderer';

 it('should render a slide caption', () => {
    const slidecaption = renderer.create(
        <SlideCaption
           url="https://www.phillips.comauctions/auction/NY010517"
           htmlCaption="Latin America"
           slideWidt="500"
        />
    );
    const tree = slidecaption.toJSON();
    expect(tree).toMatchSnapshot();
});
