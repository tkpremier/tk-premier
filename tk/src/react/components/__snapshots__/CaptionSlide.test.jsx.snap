// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a slide caption 1`] = `
<div
  className="caption-slide"
  style={
    Object {
      "width": "undefinedpx",
    }
  }
>
  <a
    href="https://www.phillips.comauctions/auction/NY010517"
    onClick={[Function]}
  >
    <div
      className="caption"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Latin America",
        }
      }
    />
  </a>
</div>
`;
